### PR TITLE
Replace wavy paths in all test files 

### PR DESCRIPTION
**Pull request by mforner13**
## Description of change

The Wavy module simplifies file paths by using ~ to represent the root folder. Due to the issues using Wavy experienced by the team, the changes in this PR remove all file paths which use Wavy and replace them with relative file paths. 

The first commit replaces instances where a file path is required using Wavy. The second commit replaces paths using wavy in proxyquire and requires proxyquire in these files. 

Proxquire was previously required globally in common.js, so requiring it in the files which use it addresses the issue that the path in prozquire is relative to where the module is imported.

## Test instructions

There should be no changes to any functionality, so testing involves making sure the tests which previously used Wavy still pass and that normal functionality remains. 

## What should I see
 
There are no visual changes. 

## TODO in future pull requests

The Wavy module itself is still present in the front-end, I need to check when this can be removed and any risks this might pose.

##Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to develop?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/investments/controllers/__test__/details.test.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
-const investmentData = require('~/test/unit/data/investment/investment-data.json')
+const investmentData = require('../../../../../test/unit/data/investment/investment-data.json')
+const proxyquire = require('proxyquire')
```
Comment (rafenden) [inline src/apps/investments/controllers/__test__/details.test.js]: We usually place imports from npm on the top of the file and local ones at the bottom.

File: src/apps/investments/middleware/forms/__test__/team-members.test.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 const { assign } = require('lodash')
 const uuid = require('uuid')
 
-const config = require('~/src/config')
-const paths = require('~/src/apps/investments/paths')
-const investmentData = require('~/test/unit/data/investment/investment-data.json')
-const { teamMembersLabels } = require('~/src/apps/investments/labels')
-const teamMembersController = require('~/src/apps/investments/middleware/forms/team-members')
+const config = require('../../../../../config/index')
```
Comment (rafenden) [inline src/apps/investments/middleware/forms/__test__/team-members.test.js]: No need to include `index` at the end.

File: src/apps/investments/middleware/forms/__test__/value.test.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-const controller = require('~/src/apps/investments/middleware/forms/value')
-const { investmentTypes } = require('~/src/apps/investments/types')
-const paths = require('~/src/apps/investments/paths')
-const config = require('~/src/config')
+const { gvaMessages } = require('../gross-value-added-message')
+const adviserData = require('../../../../../../test/unit/data/investment/interaction/advisers.json')
+const controller = require('../value')
+const { investmentTypes } = require('../../../types')
+const paths = require('../../../paths')
+const config = require('../../../../../config/index')
```
Comment (rafenden) [inline src/apps/investments/middleware/forms/__test__/value.test.js]: No need to include `index` at the end.

**Comment by peterhudec** [comment]
Seems like you have your IDE set up to autoformat array literals, which bloats te PR.
Screen Shot 2019-12-02 at 12 29 23 PM

File: src/apps/adviser/__test__/repos.test.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
-const adviserData = require('~/test/unit/data/advisers/advisers.json')
-const badAdviserData = require('~/test/unit/data/advisers/advisers-with-bad-data.json')
-const config = require('~/src/config')
-const repos = require('~/src/apps/adviser/repos')
+const adviserData = require('../../../../test/unit/data/advisers/advisers.json')
+const badAdviserData = require('../../../../test/unit/data/advisers/advisers-with-bad-data.json')
+const config = require('../../../config/index')
```
Comment (ian-leggett) [inline src/apps/adviser/__test__/repos.test.js]: We can remove index here

File: src/apps/api/__test__/services.test.js [hunk 5]
```diff
@@ -1,6 +1,8 @@
 const API_KEY = 'postcode_key'
-const services = proxyquire('~/src/apps/api/services.js', {
-  '../../config': {
+const proxyquire = require('proxyquire')
+
+const services = proxyquire('../services', {
+  '../../config/index': {
```
Comment (ian-leggett) [inline src/apps/api/__test__/services.test.js]: We can remove the index here

File: src/apps/adviser/__test__/repos.test.js (hunk 4 above)
Comment (mforner13) [reply src/apps/adviser/__test__/repos.test.js]: All changed - thanks Ian :)

File: src/apps/investments/middleware/forms/__test__/value.test.js (hunk 3 above)
Comment (mforner13) [reply src/apps/investments/middleware/forms/__test__/value.test.js]: Have removed all the indexes at the end of config - thanks Raf!

File: src/apps/investments/controllers/__test__/details.test.js (hunk 1 above)
Comment (mforner13) [reply src/apps/investments/controllers/__test__/details.test.js]: Have put proxyquire with module imports wherever it's imported - thanks!

**Pull request merged**